setup.py

```diff
@@ -55,7 +53,7 @@ def run(self):
 
 setup(
     name='sphinx_rtd_theme',
-    version=__version__,
+    version='0.4.3.dev0',
```

Comment:
I see that you're addressing this in another PR. You can also consider reading the version from `package.json`. Unfortunately, that file can't also read the version from the Python theme module (and the theme module can't read from `package.json`). I've seen a couple approaches here from reading from `package.json` to having a separate file with just the version (eg. `sphinx_rtd_theme/version.py`) to parsing the text of `sphinx_rtd_theme/__init__.py` instead of importing it. I don't have a particularly strong opinion although any of them seem preferable to hardcoding it.